ci/cloudbuild/triggers/gcc-54-ci.yaml

```diff
@@ -8,3 +8,5 @@ name: gcc-54-ci
 substitutions:
   _BUILD_NAME: gcc-54
   _DISTRO: fedora
```

Comment:
Ditto regarding fedora and gcc-5.4

Reply:
Nope. That's a bug that I introduced in #6135. I'll fix that in a followup PR.